x-pack/plugins/upgrade_assistant/__jest__/client_integration/kibana_deprecations/deprecation_details_flyout.test.ts

```diff
@@ -40,22 +40,26 @@ describe('Kibana deprecation details flyout', () => {
   });
 
   describe('Deprecation with manual steps', () => {
-    test('renders flyout with manual steps only', async () => {
+    test('renders flyout with single manual step as a standalone paragraph', async () => {
       const { find, exists, actions } = testBed;
       const manualDeprecation = mockedKibanaDeprecations[1];
 
-      await actions.table.clickDeprecationAt(1);
+      await actions.table.clickDeprecationAt(0);
 
       expect(exists('kibanaDeprecationDetails')).toBe(true);
-      expect(exists('kibanaDeprecationDetails.warningDeprecationBadge')).toBe(true);
       expect(find('kibanaDeprecationDetails.flyoutTitle').text()).toBe(manualDeprecation.title);
-      expect(find('manualStepsList').find('li').length).toEqual(
-        manualDeprecation.correctiveActions.manualSteps.length
-      );
+      expect(find('manualStep').length).toBe(1);
+    });
 
-      // Quick resolve callout and button should not display
-      expect(exists('quickResolveCallout')).toBe(false);
-      expect(exists('resolveButton')).toBe(false);
+    test('renders flyout with multiple manual steps as a list', async () => {
+      const { find, exists, actions } = testBed;
+      const manualDeprecation = mockedKibanaDeprecations[1];
+
+      await actions.table.clickDeprecationAt(1);
+
+      expect(exists('kibanaDeprecationDetails')).toBe(true);
+      expect(find('kibanaDeprecationDetails.flyoutTitle').text()).toBe(manualDeprecation.title);
+      expect(find('manualStepsListItem').length).toBe(3);
```

Comment:
I made these assertions static, because we want to be clear about our expectations. If an assertion is dynamic, based upon properties of the mock value, then it becomes difficult to understand the intention behind the assertion, and creates the possibility to inadvertently change the meaning of a test by making changes to the mock value.